### ISSUE TITLE
colorkey labeling change after package update

### ISSUE THREAD

**Issue by JLKWren** [oscarperpinan/rastervis#102]
Hi,

I just updated my rasterVis package to the newest version (0.51.5) and code I run all the time to make maps now plots differently. I need to put a title on the colorkey and I've always placed it vertical, centered, and to the right of the colorkey. After the update, the title no longer plots in this location and I haven't been able to figure out how get it to plot like it did before I updated rasterVis. I can put a title next to the bar using grid::grid.text but it's not ideal since I'm saving the images and the grid.text doesn't save in a file. But mostly, I'm wondering what has changed with the most recent rasterVis update? Is this a bug, or do you have a suggestion on how to get the title for the color bar to plot vertically, centered and to the right of the color bar?  If not, is there an old version (e.g. 0.51.2) that I can download somewhere? 

This is the code I used that placed the colorkey title vertical and to the right of the colorbar, but now places it horizontal and above the colorbar:
`levelplot(envData, pretty=T, margin=FALSE, par.setting=mapTheme, xlab=NULL, ylab=NULL, colorkey=list(title="Color title", title.gpar=list(cex=0)))`
and I have included a couple of figures (before and after) to show you the change. Same code used to run both images, only difference is the version of rasterVis (0.51.2 vs 0.51.5)

Thank you, 
Johanna

![before_update](https://user-images.githubusercontent.com/13009200/215934281-1767d26d-8f74-4c50-b3f0-b0a03abf1956.png)

![after_update](https://user-images.githubusercontent.com/13009200/215934279-3d4f3762-86d0-4af0-97ba-c492e332111a.png)

**Comment by oscarperpinan** [comment]
Hello,

The behaviour of the colorkey title was changed in order to use the new code of the `lattice::levelplot` function.  The changes were implemented in these commits (may-2022): 0aafb66a50b5f1073f44f9446b74eb5555fed93e and 6e10f55f63465cb04c21bd2a9d6efbb8ead2d785. 

You can control the position (and other parameters) of the title with `title.control` as in the next example. More details can be found in the help pages of `rasterVis::levelplot` and `lattice::levelplot`:

```
library(raster)
library(rasterVis)

f <- system.file("external/test.grd", package="raster")
     
r <- raster(f)

levelplot(r, margin = FALSE,
          colorkey = list(title = "[m]",
                          space = "right",
                          title.control = list(side = "right")))

```

**Comment by JLKWren** [comment]
Excellent, thank you for your quick help! 
~Johanna
